api-admin/src/main/java/bio/terra/pearl/api/admin/service/family/FamilyExtService.java

```diff
@@ -113,4 +117,37 @@ public List<DataChangeRecord> listChangeRecords(
 
     return familyService.findDataChangeRecordsByFamilyId(family.getId());
   }
+
+  @EnforcePortalStudyEnvPermission(permission = "participant_data_edit")
+  public Family create(PortalStudyEnvAuthContext authContext, Family family, String justification) {
+    if (!family.getStudyEnvironmentId().equals(authContext.getStudyEnvironment().getId())) {
+      throw new NotFoundException("Study environment not found");
+    }
```

Comment:
these can go either way, but I think our practice is to just set the study environment on the family to what was authed (see e.g. the portalId in SurveyExtService.create())
```suggestion
    family.setStudyEnvironmentId(authContext.getStudyEnvironment().getId());
```